### DISCUSSION TITLE
Is there a way to disable unreachable alerts via an API?

### DISCUSSION THREAD

**Discussion by pwsnla** [netdata/netdata#15835] [Q&A]
I have netdata running on a node that I shutdown on regular basis. I don't want to see unreachable alerts for that node during a scheduled shutdown but outside of the scheduled shutdown I'd like unreachable alerts to be enabled. I know that I can manually disable/enable unreachable alerts from Netdata Cloud UI but I would prefer to automate it. Is there a way to do it? 

I know that I can control/mute health alerts for a node via an API, but if I understand the documentation correctly it will not  mute unreachable alerts for that node from Netdata Cloud.

**Comment by MrZammler** [comment]
Hi @pwsnla thanks for this, will cc @car12o & @hugovalente-pm 

I know at least yes, we don't have an API for that atm.

**Comment by andrewm4894** [comment]
think this could be a nice [feature request](https://github.com/netdata/netdata/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=FEAT_REQUEST.yml&title=%5BFeat%5D%3A+) as i had another user asking about something somewhat similar where having more flexibility here like this (eg api to disable unreachable alerts on agent or something like this) would have been useful.

**Comment by andrewm4894** [comment]
@sashwathn @shyamvalsan this is also relevant for alert silencing too - ability to set up rules for silencing of unreachable alerts might be part of a solution here too - is that supported currently with alert silencing rules?

**Reply by sashwathn** [reply]
@andrewm4894 : I believe the Silencing feature would have a way to silence reachability notifications in the next iteration. @hugovalente-pm to confirm!

**Reply by hugovalente-pm** [reply]
> @andrewm4894 : I believe the Silencing feature would have a way to silence reachability notifications in the next iteration. @hugovalente-pm to confirm!

This is planned but not for the next iteration that is aimed to expose the silencing badges, that would wrap-up the first planned release
IIRC when this was discussed with @vkuznecovas and @car12o this wasn't something that could be tackled easily, that's why it was scoped out of the first release
